gini/gini-vision-lib-ios
# Finalize UI

Analysis screen UI has been improved, showing the PDF information above image instead of over, and the capture suggestions appearance animation has been modified.
Fixed also some issues.

### How to test
Import a PDF and see that the title and page count is shown above the image and an indicator with a white background is shown in the middle of the image. Also import or take a picture and see after 4 seconds that some tips appear on the bottom.

### Merging
Automatic.

### Ticket 
Issues #65, #66
